website/content/docs/v0.9/Guides/disk-encryption.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Disk Encryption"
+description: "This page contains guides related kernel level disk encryption"
+---
+
+LUKS2 disk encryption is supported by the Linux kernel modules and `cryptsetup` utility.
+
+Sequencer runs additional steps when encryption is enabled.
```

Comment:
I think `sequencer` is something internal to Talos, we never published it in the docs.

Reply:
Will rename it to system